setup.py

```diff
@@ -50,7 +50,7 @@ def readme():
 ]
 
 mypy_require = [
-    'mypy~=0.740;python_version>"3.4"',
+    'mypy~=0.740,<=0.910;python_version>"3.4"',
```

Comment:
Our code doesn't pass all the checks for newer versions of mypy, so pysoa CI is failing again. I think it's worth to limit the mypy version to make the CI work again, as it's not trivial to update the underlying code for every check (it deals with low level python like ctypes.pointer, etc.).